src/main/views/subscription-search-urn-results.njk

```diff
@@ -0,0 +1,36 @@
+{% extends "template.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "./macros/common-components.njk" import goBack, submitButton %}
+
+{% block pageTitle %}
+  {{ "Subscription URN Search Results" | safe }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ goBack('/subscription-urn-search') }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-m">Search result</h1>
```

Comment:
I think the search result heading is slightly smaller than the screenshot

Reply:
corrected